ui/appui-react/src/appui-react/widget-panels/Frontstage.tsx

```diff
@@ -299,6 +299,7 @@ export function addWidgets(state: NineZoneState, widgets: ReadonlyArray<WidgetDe
       preferredFloatingWidgetSize: widget.defaultFloatingSize,
       canPopout: widget.canPopout,
       isFloatingStateWindowResizable: widget.isFloatingStateWindowResizable,
+      hideWithUiWhenFloating: widget.hideWithUiWhenFloating ? widget.hideWithUiWhenFloating : false,
```

Comment:
Consider simplifying to: `hideWithUiWhenFloating: !!widget.hideWithUiWhenFloating`

Reply:
Done